packages/ragbits-document-search/src/ragbits/document_search/retrieval/rerankers/litellm.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+import litellm
+
+from ragbits.document_search.documents.element import Element, TextElement
+from ragbits.document_search.retrieval.rerankers.base import Reranker
+
+
+class LiteLLMReranker(Reranker):
+    """
+    A LiteLLM reranker for providers such as Cohere, Together AI, Azure AI.
```

Comment:
I think it might be helpful to describe a bit these attributes below or link the documentation, I couldn't find `litellm`, but `cohere` is probably the same https://docs.cohere.com/v2/reference/rerank#request